src/test/rustdoc-ui/invalid-syntax.stderr

```diff
@@ -65,7 +65,8 @@ warning: could not parse code block as Rust code
 LL |   ///     code with bad syntax
    |  _________^
 LL | | ///     \_
-   | |__________^
+LL | | ///
+   | |_
```

Comment:
This is a diagnostic regression.

Reply:
I'm actually not able to understand the original behavior. The first line is supposed to be the one triggering the error so I don't understand why we had an arrow for the second as well.

Reply:
The span is intended to contain the codeblock in the markdown exactly. Here, it includes the first `/` of the next line of the doc comment.

Reply:
Oh I see! Great catch!